<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Utterances:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QLineEdit" name="lineeditSearchUtterances">
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Words:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QLineEdit" name="lineeditSearchWords">
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Morphemes:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="3">
       <widget class="QLineEdit" name="lineeditSearchMorphemes">
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Glosses:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="3">
       <widget class="QLineEdit" name="lineeditSearchGlosses">
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Translations:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="3">
       <widget class="QLineEdit" name="lineSearchTranslations">
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Files:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QLabel" name="label_8">
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="2">
       <widget class="QListWidget" name="listFiles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="3" colspan="2">
       <widget class="PoioIlTextEdit" name="texteditInterlinear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QPushButton" name="buttonAddFiles">
        <property name="text">
         <string>Add files...</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QPushButton" name="buttonRemoveFiles">
        <property name="text">
         <string>Remove files</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="buttonSaveSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Save Search...</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="buttonSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="sizeIncrement">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiel">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddFiles"/>
    <addaction name="actionRemoveFiles"/>
    <addaction name="separator"/>
    <addaction name="actionEditWordClasses"/>
    <addaction name="separator"/>
    <addaction name="actionSearches"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutPoioAnalyzer"/>
   </widget>
   <addaction name="menuFiel"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="text">
    <string>Add Files...</string>
   </property>
  </action>
  <action name="actionRemoveFiles">
   <property name="text">
    <string>Remove Files</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionEditWordClasses">
   <property name="text">
    <string>Edit Word Classes...</string>
   </property>
  </action>
  <action name="actionSearches">
   <property name="text">
    <string>Searches...</string>
   </property>
  </action>
  <action name="actionAboutPoioAnalyzer">
   <property name="text">
    <string>About PoioAnalyzer...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PoioIlTextEdit</class>
   <extends>QTextEdit</extends>
   <header>PoioIlTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
